lib/Connectable.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
+    /** 
```

Comment:
:no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).